presto-iceberg/src/main/java/com/facebook/presto/iceberg/TableStatisticsMaker.java

```diff
@@ -459,6 +522,10 @@ public static List<ColumnStatisticMetadata> getSupportedColumnStatistics(String
             supportedStatistics.add(NUMBER_OF_DISTINCT_VALUES.getColumnStatisticMetadataWithCustomFunction(columnName, "sketch_theta"));
         }
 
+        if (!(type instanceof FixedWidthType)) {
```

Comment:
Nice, I think this is clearer than before now about what types we track the size for